src/Processors/Formats/Impl/JSONEachRowRowInputFormat.cpp

```diff
@@ -360,8 +360,30 @@ void registerInputFormatJSONEachRow(FormatFactory & factory)
         return std::make_shared<JSONEachRowRowInputFormat>(buf, sample, std::move(params), settings, false);
     });
 
+    factory.registerInputFormat("JSONLines", [](
```

Comment:
Does the factory allow registration of aliases?

Reply:
I checked it does not support registration of aliases.